cysimdjson/cysimdjson.pyx

```diff
@@ -186,7 +224,7 @@ cdef class JSONElement:
 		cdef simdjson_object.iterator it = obj.begin()
 		while it != obj.end():
 			sv = it.key()
-			yield string_view_to_string(sv)
+			yield string_view_to_string(sv).decode("utf-8")
```

Comment:
`.decode("utf-8")` represent unneeded performance penalty here ... extend `string_view_to_string()` implementation b/c we know that the underlaying string is UTF-8 so we can use unchecked Python string constructor.